dev/com.ibm.ws.jaxrs.2.0_fat/publish/servers/com.ibm.ws.jaxrs.fat.annotationscan/bootstrap.properties

```diff
@@ -0,0 +1,6 @@
+com.ibm.ws.logging.trace.specification=*=info:\
+com.ibm.ws.jaxrs20.*=all:\
```

Comment:
I'm fine with this as-is, but are we going to get into trouble for filling the build machines with extra trace?

Reply:
you are right, I tried cleaning up the ones I touched but I should make a pass on all of these servers and clean the trace up to lower our trace footprint.